packages/lib/server/repository/workflow.ts

```diff
@@ -276,4 +285,114 @@ export class WorkflowRepository {
       };
     }
   }
+  static async getRemindersFromRemovedTeams(
```

Comment:
These methods have been copied and pasted from https://github.com/calcom/cal.com/pull/16861/files#diff-f6ef6880041c2afa0023a8b231030af6fc89b5c067f3920efa906890c100734f